src/host/outputStream.cpp

```diff
@@ -76,8 +76,13 @@ void WriteBuffer::_DefaultStringCase(const std::wstring_view string)
 {
     size_t dwNumBytes = string.size() * sizeof(wchar_t);
 
-    _io.GetActiveOutputBuffer().GetTextBuffer().GetCursor().SetIsOn(true);
+    Cursor& cursor = _io.GetActiveOutputBuffer().GetTextBuffer().GetCursor();
+    if (!cursor.IsOn())

```

Comment:
Here `SetIsOn()` calls `RedrawCursorAlways()` and causes unnecessary overhead.

I feel this one is safe, since we deliberately want the cursor to be on and visible.